shub/apps/library/views/images.py

```diff
@@ -551,6 +554,60 @@ def get(self, request):
         collections = generate_collections_list(token.user)
         return Response(data=collections, status=200)
 
+    def post(self, request):
+        """Create a new collection.
+
+        POST /v1/collections
+
+        Body parameters:
+        * entity: entity id (= user id)
+        * name: new collection name
+        * private: optional boolean (defaults to the configured default for
+          new collections
+
+        Return the newly created collection.
+        """
+
+        print("POST CollectionsView")
+        if not validate_token(request):
+            print("Token not valid")
+            return Response(status=403)
+
+        # body should have {'entity': entity_id, 'name': new_collection_name, 'private': bool}
+        # 'private' is optional and defaults to Collection.private default
+        # value set with get_privacy_default()
+        # {"entity": "42", "name": "my_collection", "private": true}
+        body = json.loads(request.body.decode("utf-8"))
+        if not ("entity" in body and "name" in body):
+            print("Invalid payload")
+            return Response(status=400)
+
+        # does a collection with the same name exist already?
+        if not get_collection(body["name"], False) is None:
```

Comment:
You don't have to say `is None` you can just say `if not X`.

Reply:
OK.